manifests/pcap-api.yml

```diff
@@ -39,15 +39,34 @@ instance_groups:
         release: pcap
         properties:
           pcap-api:
-            agent_tls_skip_verify: true
-            enable_server_tls: false
-            cf_api: ((pcap_api.cf_api))
-            bosh_director_api: ((pcap_api.bosh_director_api))
-            bosh_director_ca: ((pcap_api.bosh_director_ca))
-            client_cert: ((pcap_api_mtls.certificate))
-            client_key: ((pcap_api_mtls.private_key))
-            agent_ca: ((pcap_api_mtls.ca))
-            agent_common_name: pcap-agent.service.cf.internal
+            log_level: info
+            buffer:
+              size: 1000
+              upper_limit: 995
+              lower_limit: 900
+            concurrent_captures: 5
+            listen:
+              port: 8080
+            #              tls:
+            #                certificate: ((pcap_api_platform_mtls.certificate))
+            #                private_key: ((pcap_api_platform_mtls.private_key))
+            #                ca: ((/bosh-aws-cfn07/cf/router_ca.ca))
+            bosh:
+              director_url: ((pcap_api.bosh_director_api))
+              token_scope: bosh.admin
+              agent_port: 9494
+              mtls:
+                common_name: bosh.service.cf.internal
+                skip_verify: false
+                certificate: ((pcap_api_bosh_mtls.certificate))
+                private_key: ((pcap_api_bosh_mtls.private_key))
+                ca: ((/bootstrap-bosh-aws-cfn07/bosh/default_ca.ca))
```

Comment:
Please adapt the path for the ca property

Reply:
Will be done as part of a followup.